src/dxvk/dxvk_adapter.cpp

```diff
@@ -25,12 +25,26 @@ namespace dxvk {
   : m_instance      (&instance),
     m_handle        (handle),
     m_capabilities  (instance, handle, nullptr) {
+    const auto& properties = m_capabilities.getProperties();
+
+    if (properties.vk11.deviceLUIDValid) {
```

Comment:
Should probably gate this behind a wine / winevulkan check? Not sure if this is going to break Windows or not, but I suppose this whole thing isn't going to work there anyway.

Reply:
Every D3DKMT call introduced here should work as intended on Windows, except for the D3DKMTEscape calls which are Wine-specific. This means that importing a shared resource should work on Windows, but creating and exporting one wouldn't as there's no way to associate the resource runtime descriptor.

This only the case for resources and importing and exporting fences should work in both ways. Also, on Windows with NVIDIA drivers I believe a resource runtime descriptor is sometimes automatically being created somehow by the Vulkan driver, so exporting might work in that case as well.